ThingIFSDK/ThingIFSDK/TriggeredCommandForm.swift

```diff
@@ -47,32 +47,73 @@ open class TriggeredCommandForm: NSObject, NSCoding {
     /// Meta data of ad command.
     open let metadata: Dictionary<String, Any>?
 
+    /// Use trait or not.
+    open let useTrait: Bool
```

Comment:
I think we should not open this attribute, since the init with useTrait is made with private method.